NorthstarDLL/serverchathooks.cpp

```diff
@@ -37,31 +37,27 @@ void(__fastcall* MessageWriteBool)(bool bValue);
 bool bShouldCallSayTextHook = false;
 // clang-format off
 AUTOHOOK(_CServerGameDLL__OnReceivedSayTextMessage, server.dll + 0x1595C0,
-void, __fastcall, (CServerGameDLL* self, unsigned int nSenderPlayerIndex, const char* text, bool isTeam))
+void, __fastcall, (CServerGameDLL* self, unsigned int senderPlayerId, const char* text, bool isTeam))
```

Comment:
why was this renamed? not necessary and arguably worse name than before

Reply:
previously thought this was Index, but turns out it's actually ID